src/formpack/reporting/export.py

```diff
@@ -734,13 +729,17 @@ def to_xlsx(self, filename, submissions):
         sheet_row_positions = defaultdict(lambda: 0)
 
         def _append_row_to_sheet(sheet_, data):
+            # Ensure all list objects are coerced to strings otherwise
+            # xlswriter will fail to export
+            data_ = [str(d) if isinstance(d, list) else d for d in data]
```

Comment:
@joshuaberetta, is `_notes` also a list? I thought it was a dictionary (but maybe it was a list of dictionaries?). I wonder if it'd be better to unconditionally `str()` everything at this point and add a TODO comment indicating that it'll have to change when we tackle #44 and #59. (#58 is also related, but I don't understand that issue since it contradicts #44).

Could `_submission__notes` also crop up? I think all `copy_fields` are propagated to all sections by #169 when repeating groups are used. I'm going to have a look at the field classes in a minute to see if there's a cleaner way to manage this.

When this is done, we can close #65 thanks to your work.

Reply:
@jnm `_notes` pops up as a list of dictionaries so instead of handling it separately I figured that line will handle both the case of `_notes` and `_submission__notes`. Since `_tags` and `_submission__tags` is being joined into a comma-separated string, I thought it could still be handled on its own.

Reply:
Olivier and I found this issue while I was trying to sort this out: https://github.com/kobotoolbox/kobocat/issues/690. Clearly not m?any people add tags to submissions.

Reply:
OK, thanks. I'll merge this now and will look into the field classes afterwards.

Reply:
@dorey, it looks like this was buttoned up nicely in https://github.com/kobotoolbox/formpack/pull/244/files#diff-a288042dcc6cd2da3626a48b8620c0e256dcb606dd52aa762c5c6c0890d58ae2R673-R710, so maybe this isn't the right place to be looking for the source of the `Unsupported type <class 'list'> in write()` error.
Internal discussion: https://chat.kobotoolbox.org/#narrow/stream/4-KoBo-Dev/topic/formpack.20error/near/52661.